.github/workflows/publish.yml

```diff
@@ -29,16 +29,61 @@ jobs:
         #output: versionType, fullVersion
         #fails if versionType is BAD, which interrupts the workflow
         run: ./gradlew qualifyVersionGha
-      - name: run checks
-        id: checks
-        run: ./gradlew check
+      - name: run reactor-netty-core checks
+        id: reactor-netty-core-checks
+        run: ./gradlew reactor-netty-core:check
+  reactor-netty-http:
+    # Notes on reactor-netty-http: this job has no access to secrets, only github token. As a result, all non-core actions are centralized here
+    # This includes the tagging and drafting of release notes. Still, when possible we favor plain run of gradle tasks
```

Comment:
nit: these two lines of comments are not entirely correct anymore. could keep only the first sentence

Reply:
fixed